src/api/python/CMakeLists.txt

```
@@ -39,4 +39,39 @@ add_library(pycvc4
 target_link_libraries(pycvc4 cvc4 ${PYTHON_LIBRARIES})
 
 python_extension_module(pycvc4)
-install(TARGETS pycvc4 DESTINATION lib)
+
+# Installation based on https://bloerg.net/2012/11/10/cmake-and-distutils.html
+# Create a wrapper python directory and generate a distutils setup.py file
+set(SETUP_PY_IN "${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in")
+set(SETUP_PY    "${CMAKE_CURRENT_BINARY_DIR}/setup.py")
+configure_file(${SETUP_PY_IN} ${SETUP_PY})
```

You want setup.py to be in `build/src/api/python`
```suggestion
configure_file(setup.py.in setup.py)
```

---

Made that change with aeeb6614682a8fe0924880e592b42a16d7a42544. Also had to update the setup.py in the install command or it couldn't find it.